checkov/terraform/checks/resource/azure/AppServiceSlotMinTLS.py

```diff
@@ -1,10 +1,12 @@
+from typing import Any, List
+
 from checkov.common.models.enums import CheckCategories, CheckResult
 from checkov.terraform.checks.resource.base_resource_value_check import BaseResourceValueCheck
 
 
 class AppServiceSlotMinTLS(BaseResourceValueCheck):
     def __init__(self):
-        name = "Ensure the App service slot is using the latest version of TLS encryption"
+        name = "Ensure web app slot uses at least TLS 1.2"
```

Comment:
```suggestion
        name = "Ensure the App service slot is using the latest version of TLS encryption"
```

Reply:
Thanks. Fixed in [70a8eaa](https://github.com/bridgecrewio/checkov/pull/6956/commits/70a8eaaad1754050a139710a54b03b8a1b05da64).